WireUI/Sources/WireAccountImageUI/AccountImageView/AccountImageView.swift

```diff
@@ -93,12 +93,16 @@ public final class AccountImageView: UIView {
         set { availabilityIndicatorView.backgroundViewColor = newValue }
     }
 
+    public var notifications: ProfileNotifications? {
```

Comment:
Why not just a simple Bool like `isNotificationBadgeVisible`?

Reply:
replaced